src/infrastructure/remote/neoga.ts

```diff
@@ -131,6 +131,49 @@ export function NeogaDataRemote(): NeogaService {
     else throw '서버 통신 실패';
   };
 
+  const getMostFormCard = async () => {
+    const response = await privateAPI.get({ url: `/form/most` });
+    if (response.status === STATUS_CODE.OK)
+      return {
+        resultList: response.data.resultList
+          ? response.data.resultList.map((result: any) => ({
+              id: result.id,
+              title: result.title.replace('\\n', '\n'),
+              subtitle: result.subtitle,
+              darkIconImage: result.darkIconImage,
+              createdAt: result.createdAt,
+              answer: result.answer
+                ? result.answer.map((comment: any) => ({
+                    id: comment.id,
+                    name: comment.name,
+                    relationship: comment.relationship,
+                    content: comment.content,
+                    keyword: comment.keyword
+                      ? comment.keyword.map((word: any) => ({
+                          id: word.id,
+                          content: word.name,
+                          color: word.colorCode,
+                          fontColor: word.fontcolor,
+                        }))
+                      : [],
+                  }))
+                : [],
+            }))
+          : response.data
+          ? response.data.map((result: any) => ({
+              id: result.id,
+              title: result.title.replace('\\n', '\n'),
+              subtitle: result.subtitle,
+              darkIconImage: result.darkIconImage,
+              createdAt: result.createdAt,
+              answer: [],
+            }))
+          : [],
```

Comment:
api 명세서 보니까 response.data.resultList가 없으면서 response.data는 없는 경우는 없어보이는데 그러면 여기는 고냥 빈 배열만 줘도 되지 않을까요??
```suggestion
          : [],
```

Reply:
그리고 개인적으로 삼항연산자가 이정도로 복잡해지면 if문을 쓰는 것도 괜찮지 않나 생각합니다~!

Reply:
이 부분 전에 PR #163 의 '⛑hotfix: 상세 페이지에서 답변이 아무것도 없을 때 안 보이는 오류 수정' 커밋에서 추가됐던 부분인데, @SeojinSeojin 혹시 어떤 점에서 이 부분이 필요했는지 알려주실 수 있나요? 주영 언니가 말해준 대로 빈 배열만 줘봤는데 오류가 안 나서 궁금합니다 !!